view/script/trial.js

```diff
@@ -17,7 +17,26 @@
         }]
     };
     var ctx = document.getElementById('trialChart').getContext('2d');
+    var differenceValue = 0;
 
+    document.getElementById('toDate').setAttribute('min', document.getElementById('fromDate').value);
+    $('#fromDate').on('change', function changeToDateValue () {
+        var fromDateValue = document.getElementById('fromDate').value;
+        var newtoDateValue = moment(fromDateValue).add(differenceValue, 'days').format('YYYY-MM-DD');
+
+        document.getElementById('toDate').value = newtoDateValue;
+    }).on('focus', function storeDateDiff (event) {
+        var oldfromDateValue = event.target.value;
+        var oldtoDateValue = document.getElementById('toDate').value;
+
+        differenceValue = moment(oldtoDateValue).diff(moment(oldfromDateValue), 'days');
```

Comment:
the differenceValue which is calculated here, where is it again being used?
On line 25 I can see that the newtoDateValue is being calculated and there initalized value of differenceValue  is being used.

Could you please provide an explanation for this part of code